examples/python/data_transmission/README.cn.md

```diff
@@ -6,31 +6,31 @@
 
   在一次调用中, 客户端只能向服务器传输一次请求数据, 服务器也只能返回一次响应
 
-  `client.py - line:14 - simple_method`
+  `client.py : simple_method`
 
-  `server.py - line:17 - SimpleMethod`
+  `server.py : SimpleMethod`
```

Comment:
Can we either change the formating to:
```Markdown
* `client.py: simple_method`
* `server.py: SimpleMethod`
```
or
```Markdown
* `client.simple_method`
* `server.DemoServer.SimpleMethod`
```

Apply to all other code reference.

Reply:
I chose the first style.